### PR TITLE
Cover missing parts of pkg/mount/block

### PR DESCRIPTION
**Pull request by jensdrenhaus**

### CODE REVIEW

**Comment by jensdrenhaus** [comment]
If someone has ideas to cover the remaining open error paths, I'm all ears.

File: pkg/mount/block/blockdev_test.go [hunk 1]
```diff
@@ -6,33 +6,259 @@ package block
...
 
-// TestFindMountPointNotExists checks that non existent
-// entry is checked and nil returned
-func TestFindMountPointNotExists(t *testing.T) {
-	LinuxMountsPath = "testdata/mounts"
-	if _, err := GetMountpointByDevice("/dev/mapper/sys-oldxxxxxx"); err == nil {
-		t.Errorf(`GetMountpointByDevice("/dev/mapper/sys-oldxxxxxx") = _, %v, want not nil`, err)
+func TestDebug(t *testing.T) {
+	Debug("foo")
+}
```
Comment (ChriMarMe) [inline pkg/mount/block/blockdev_test.go:20]: I cant point my finger on it, but somehow I'm unhappy with this. Does this add any value?

File: pkg/mount/block/blockdev_test.go [hunk 2]
```diff
@@ -6,33 +6,259 @@ package block
...
+
+func TestBlockDevDevicePath(t *testing.T) {
+	name := "devname"
+	b := &BlockDev{Name: name}
+	want := filepath.Join("/dev", name)
+	got := b.DevicePath()
+	if got != want {
+		t.Errorf("got %s, want %s", got, want)
+	}
```
Comment (ChriMarMe) [inline pkg/mount/block/blockdev_test.go:80]: Error message style

File: pkg/mount/block/blockdev_test.go [hunk 3]
```diff
@@ -6,33 +6,259 @@ package block
...
+}
+
+func TestBlockDevDeviceName(t *testing.T) {
+	want := "devname"
+	b := &BlockDev{Name: want}
+	got := b.DevName()
+	if got != want {
+		t.Errorf("got %s, want %s", got, want)
+	}
```
Comment (ChriMarMe) [inline pkg/mount/block/blockdev_test.go:89]: Error message style

File: pkg/mount/block/blockdev_test.go [hunk 4]
```diff
@@ -6,33 +6,259 @@ package block
...
+			dev:  &BlockDev{Name: "noexist"},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			_, err := tt.dev.GPTTable()
+			if err == nil {
+				t.Error("Expect an error")
+			}
```
Comment (ChriMarMe) [inline pkg/mount/block/blockdev_test.go:113]: Which errors are expected?

File: pkg/mount/block/blockdev_test.go [hunk 5]
```diff
@@ -6,33 +6,259 @@ package block
...
+	}
+}
+
+func TestBlockDevPhysicalBlockSizeError(t *testing.T) {
+	// Success paths are covered in vm_test.go
+	dev := &BlockDev{Name: "noexist"}
+	_, err := dev.PhysicalBlockSize()
+	if err == nil {
+		t.Error("Expect an error")
+	}
```
Comment (ChriMarMe) [inline pkg/mount/block/blockdev_test.go:124]: Which errors are expected?

File: pkg/mount/block/blockdev_test.go [hunk 6]
```diff
@@ -6,33 +6,259 @@ package block
...
+}
+
+func TestBlockDevBlockSizeError(t *testing.T) {
+	// Success paths are covered in vm_test.go
+	dev := &BlockDev{Name: "noexist"}
+	_, err := dev.BlockSize()
+	if err == nil {
+		t.Error("Expect an error")
+	}
```
Comment (ChriMarMe) [inline pkg/mount/block/blockdev_test.go:133]: Which errors are expected?

File: pkg/mount/block/blockdev_test.go [hunk 7]
```diff
@@ -6,33 +6,259 @@ package block
...
+}
+
+func TestBlockDevKernelBlockSizeError(t *testing.T) {
+	// Success paths are covered in vm_test.go
+	dev := &BlockDev{Name: "noexist"}
+	_, err := dev.KernelBlockSize()
+	if err == nil {
+		t.Error("Expect an error")
+	}
```
Comment (ChriMarMe) [inline pkg/mount/block/blockdev_test.go:142]: Which errors are expected?

File: pkg/mount/block/blockdev_test.go [hunk 8]
```diff
@@ -6,33 +6,259 @@ package block
...
+			dev:  &BlockDev{Name: "noexist"},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			_, err := tt.dev.Size()
+			if err == nil {
+				t.Error("Expect an error")
+			}
```
Comment (ChriMarMe) [inline pkg/mount/block/blockdev_test.go:166]: Which errors are expected?

File: pkg/mount/block/blockdev_test.go [hunk 9]
```diff
@@ -6,33 +6,259 @@ package block
...
+	}
+}
+
+func TestBlockDevPCIInfoError(t *testing.T) {
+	// Success paths are covered in vm_test.go
+	dev := &BlockDev{Name: "noexist"}
+	_, err := dev.PCIInfo()
+	if err == nil {
+		t.Error("Expect an error")
+	}
```
Comment (ChriMarMe) [inline pkg/mount/block/blockdev_test.go:186]: Which errors are expected?

**Review by ChriMarMe — COMMENTED** [review]
LGTM!

Just some smoll nits about some error checking.

File: pkg/mount/block/blockdev_test.go (hunk 1 above)
Comment (jensdrenhaus) [reply pkg/mount/block/blockdev_test.go:20]: Same here, it just covers
``` golang
// Debug function to override for verbose logging.
Debug = func(string, ...interface{}) {}
```
in blockdev_linux.go

File: pkg/mount/block/blockdev_test.go (hunk 4 above)
Comment (jensdrenhaus) [reply pkg/mount/block/blockdev_test.go:113]: I think it is not necessary to specify further here. This error message together with the name of the subtest should give enough context in the test log. 
What would be your suggestion for the message?

File: pkg/mount/block/blockdev_test.go (hunk 5 above)
Comment (jensdrenhaus) [reply pkg/mount/block/blockdev_test.go:124]: See above.

File: pkg/mount/block/blockdev_test.go (hunk 6 above)
Comment (jensdrenhaus) [reply pkg/mount/block/blockdev_test.go:133]: See above.

File: pkg/mount/block/blockdev_test.go (hunk 8 above)
Comment (jensdrenhaus) [reply pkg/mount/block/blockdev_test.go:166]: See above.

File: pkg/mount/block/blockdev_test.go [hunk 10]
```diff
@@ -6,33 +6,259 @@ package block
...
+}
+
+func TestBlockDevReadPartitionTableError(t *testing.T) {
+	// Success paths are covered in vm_test.go
+	dev := &BlockDev{Name: "noexist"}
+	err := dev.ReadPartitionTable()
+	if err == nil {
+		t.Error("Expect an error")
+	}
```
[the remark this answers is not in the document]
Comment (jensdrenhaus) [reply pkg/mount/block/blockdev_test.go:177]: See above.

File: pkg/mount/block/blockdev_test.go (hunk 9 above)
Comment (jensdrenhaus) [reply pkg/mount/block/blockdev_test.go:186]: See above.

**Comment by ChriMarMe** [comment]
Squash all commits to one and we can merge

**Comment by ChriMarMe** [comment]
CodeCov does CodeCov things. Base commit got lost somewhere on the journey. ¯\_(ツ)_/¯

**Review by walterchris — APPROVED** [review]

**Pull request merged**